README.md

```diff
@@ -23,6 +24,9 @@ OpenSearch Java Client
 For more information, see [opensearch.org](https://opensearch.org/).
 This client is meant to replace the existing [OpenSearch Java High Level REST Client](https://opensearch.org/docs/latest/clients/java-rest-high-level/).
 
+## Sample code
```

Comment:
Nit: Make it capital Code to match other titles, so "Sample Code" both in the TOC and in the actual section.

Reply:
> Nit: Make it capital Code to match other titles, so "Sample Code" both in the TOC and in the actual section.

Done.